components/create_post/create_post.jsx

```diff
@@ -763,12 +763,19 @@ class CreatePost extends React.PureComponent {
 
         e.preventDefault();
 
-        let message = '';
+        let message = this.state.message;
         if (isGitHubCodeBlock(table.className)) {
-            message = '```\n' + getPlainText(e.clipboardData) + '\n```';
-        } else {
-            message = formatMarkdownTableMessage(table, this.state.message.trim());
+            const codeBlock = '```\n' + getPlainText(e.clipboardData) + '\n```';
```

Comment:
not that it would be a huge gain in processing, but I would have put both in the same function to make sure it is treated the same way in both places

Reply:
Done